build.html

```diff
@@ -3,4 +3,150 @@
 title: Building Your Application
 layout: default
 ---
-<h1>Building Your Application</h1>
\ No newline at end of file
+<main class="ds-base ds-l-container">
+  <a class="ds-c-skip-nav" href="#content">Skip to main content</a>
+  <div class="ds-l-row ds-u-fill--primary-darker">
+    <div class="ds-l-col--12 ds-u-text-align--right ds-u-font-size--display ds-u-color--white">
+      Building Your Application
+    </div>
+    <div class="ds-l-col--12 ds-u-text-align--right ds-u-font-size--h1 ds-u-color--white">
+      The User Guide for ACO's in Production
```

Comment:
This user guide will wind up being for ACOs in prod and sbx; it's written as though for prod, because the actions are the same, but I want to make clear in our headers that this user guide applies to building applications in both environments.